deps/spidershim/src/v8function.cc

```diff
@@ -27,6 +27,32 @@
 
 namespace v8 {
 
+MaybeLocal<Object>
+Function::NewInstance(Local<Context> context,
+                      int argc, Handle<Value> argv[]) const
+{
+  Isolate* isolate = Isolate::GetCurrent();
+  JSContext* cx = JSContextFromIsolate(isolate);
+  JS::RootedObject thisObj(cx, &reinterpret_cast<const JS::Value*>(this)->toObject());
```

Comment:
> JS_New requires a handle, is there some other way to get a handle here?
> 
> Ah, I missed the JS_New call. I don't know of another way to get a handle here, so never mind. ;-)

Yeah, this is the reason for a lot of things in spidershim to be Rooted where they didn't need to except because of this...